objects/Camera.js

```diff
@@ -14,7 +14,6 @@ class Camera {
     }
 
     static fromXML(XMLObject) {
-        // TODO: replace ip
```

Comment:
You removed this, but shouldn't the ip field be removed, too? We do and will not use this anymore since we will not use the ip's of the cameras.